clang-tools-extra/clangd/index/remote/Index.proto

```diff
@@ -131,3 +131,21 @@ message Relation {
   optional string subject_id = 1;
   optional Symbol object = 2;
 }
+
+message ContainedRefsRequest {
+  required string id = 1;
```

Comment:
sorry I missed the `required` bits here and below. it's [discouraged](https://protobuf.dev/programming-guides/dos-donts/#add-required) and we're having troubles consuming this internally now.

there's no particular reason for these fields to be `required`, we're already taking care of error-handling during marshalling.

my colleague is going to revert this, I am happy to land it tomorrow with relevant changes. sorry for the second back and forth @HighCommander4  :/

Reply:
No worries, please go ahead